platforms/core-runtime/logging/src/main/java/org/gradle/internal/logging/DefaultLoggingConfiguration.java

```diff
@@ -28,7 +29,15 @@
 public class DefaultLoggingConfiguration implements Serializable, LoggingConfiguration {
     private LogLevel logLevel = LogLevel.LIFECYCLE;
     private ShowStacktrace showStacktrace = ShowStacktrace.INTERNAL_EXCEPTIONS;
-    private ConsoleOutput consoleOutput = ConsoleOutput.Auto;
+    private ConsoleOutput consoleOutput = getDefaultConsoleOutput();
+
+    private static ConsoleOutput getDefaultConsoleOutput() {
+        if ("windows-aarch64".equals(Platform.current().getId())) {
```

Comment:
Aaaarggh.... Maybe making use of `org.gradle.platform.internal.CurrentBuildPlatform` or something typesafe like that would be better?